great-idea/index.html

```diff
@@ -13,11 +14,88 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
+<div class="container">
```

Comment:
Be careful here, you opened this `<div>` outside your `<body>` (and closed inside it).
Everything that got rendered in your page should go inside the body and there shouldn't be anything between your `<head>` and your `<body>`